docs/content/en/install/doks/_index.md

```diff
@@ -0,0 +1,332 @@
+
+```console
+# NOTE your kubeconfig command WILL BE DIFFERENT, retrieve the command from the DO console interface.
+doctl kubernetes cluster kubeconfig save REDACTED
+```
+
+**Output**:
+
+```bash
+Notice: Adding cluster credentials to kubeconfig file found in "~/.kube/config"
+Notice: Setting current-context to do-nyc3-k8s-1-20-7-do-0-nyc3-1623165482131
+```
+
+```bash
+kubectl cluster-info
+```
+
+**Output**:
+
+```bash
+Kubernetes control plane is running at https://f10bb874-562f-4328-8910-e0e2f6efa78b.k8s.ondigitalocean.com
+CoreDNS is running at https://f10bb874-562f-4328-8910-e0e2f6efa78b.k8s.ondigitalocean.com/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+```bash
+kubectl version
+```
+
+**Output**:
+
+```bash
+Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:18:45Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}
+Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.7", GitCommit:"132a687512d7fb058d0f5890f07d4121b3f0a2e2", GitTreeState:"clean", BuildDate:"2021-05-12T12:32:49Z", GoVersion:"go1.15.12", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+### Taint Nodes
+
+With mixed node types available we should apply taints to each node type in order to facilitate scheduling. The larger instances will be tainted with the value `app=cassandra:NoSchedule` effectively blocking the scheduling of Pods unless they have the taint `app=cassandra`. In our example deployment here Cassandra nodes are part of the pool named `pool-hluacyqq4`.
+
+```console
+# Taint Cassandra nodes
+kubectl taint node -l doks.digitalocean.com/node-pool=pool-hluacyqq4 app=cassandra:NoSchedule
+```
+
+**Output:**
+
+```console
+node/pool-hluacyqq4-8edt2 tainted
+node/pool-hluacyqq4-8edtl tainted
+node/pool-hluacyqq4-8edtt tainted
+```
+
+{{% alert title="Important" color="primary" %}}
+Your node pool names will most likely not match the values above. Make sure you adjust the commands / values appropriately for your environment.
+{{% /alert %}}
+
+## Install K8ssandra
+
+With all of the infrastructure provisioned we can now focus on installing K8ssandra. This will require configuring a service account for the backup and restore service and creating a set of Helm variable overrides.
+
+### Create Backup / Restore Service Account Secrets
+
+In order to allow for backup and restore operations, we must create a service account for the Medusa operator which handles coordinating the movement of data to and from DigitalOcean Spaces. As part of the provisioning section a key was generated for this purpose. Plug in the key and secret from the provisioning section in the following file.
+
+{{< readfilerel file="spaces.secret.yaml" highlight="yaml" >}}
```

Comment:
Because I didn't save the secret when I first created the service account and navigated somewhere else in the DO console, I could not access the secret and had to hit the "regenerate" button. Maybe suggest saving that information off right away, or reorder things so that you don't generate the secret until you need it.

Reply:
Added a note that we should write it down for usage later. Also indicated if you don't have it then you can regenerate.